Maple2.Server.Game/Util/SkillUtils.cs

```diff
@@ -119,4 +139,13 @@ public static class SkillUtils {
 
         return true;
     }
+
+    private static bool Check(this BeginConditionWeapon weapon, FieldPlayer player) {
+        bool IsValid(int handType, EquipSlot slot) {
```

Comment:
nit: can you move this function below the return statement (also with a line break between) so it's obvious that this is a function and not code being run when the function is called.